src/msr_data_skx.cpp

```diff
@@ -490,7 +498,8 @@ namespace geopm
                     "units":     "joules",
                     "scalar":    1.5258789063e-05,
                     "writeable": false,
-                    "aggregation": "sum"
+                    "aggregation": "sum",
+                    "description": "Actual energy used."
```

Comment:
increasing meter of energy used over time. It will reset periodically due to roll-over.

Reply:
... used of the DRAM over time. ...